rmills/bin/man2html.sh

```diff
@@ -103,7 +103,7 @@ $NoDepage  = 0;		# Do not strip page information
 $NoHeads   = 0;		# Do no header detection flag
 $SeeAlso   = 0;		# Do only SEE ALSO xrefs flag
 $Solaris   = 0;		# Solaris keyword search processing flag
-$Sun       = 0;		# Headers not overstriken flag
+$Sun       = 0;		# Headers no overstrike flag
```

Comment:
`overstrike` would normally conjugate just like `strike`.

`strike` -> `stricken`

I'm not sure what this line is trying to say, but `overstricken` is a bit of an awkward word, so I hoped I could avoid it.